app/assets/javascripts/components/header/header.es6.jsx

```diff
@@ -3,12 +3,14 @@ class Header extends Component {
   static get propTypes() {
     return {
       handleClick: React.PropTypes.func.isRequired,
```

Comment:
Can you change the `handleClick` prop to be called `toggleSidebar`, and set it to be *not* required?
It should be not required because pages that don't render the sidebar won't pass it anything.
You'll also need to change `handleClick` to `toggleSidebar` in parent components that render the `Header` component!